packages/batch/tests/unit/asyncProcessPartialResponse.test.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Test processPartialResponse function
+ *
+ * @group unit/batch/function/processpartialresponse
```

Comment:
Same as comment for `asyncBatchProcessor` tests -- do we need tests for Lambda context here?

Reply:
Added!